torch/csrc/autograd/saved_variable.h

```diff
@@ -52,5 +55,6 @@ class TORCH_API SavedVariable {
   bool was_default_constructed_ = true;
   bool requires_grad_ = false;
   bool has_grad_fn_ = false;
+  bool is_inplace_view_ = false;
```

Comment:
We had a lot of good discussion in the PR, and it would be very helpful this discussion was replicated in a more durable form in the source code itself, in case someone is reading the code in question.